app/build.gradle

```diff
@@ -223,6 +204,12 @@ dependencies {
 
     //Timber
     implementation 'com.jakewharton.timber:timber:5.0.1'
+
+    //Analytics
+    analyticsImplementation 'com.google.android.play:core:1.10.2'
+    analyticsImplementation 'com.google.firebase:firebase-analytics:20.0.0'
+    analyticsImplementation 'com.mixpanel.android:mixpanel-android:5.8.4'
+    analyticsImplementation 'com.google.firebase:firebase-crashlytics:18.2.5'
 }
```

Comment:
I don't think we want to put these here. The idea behind a 'noAnalytics' flavour was specifically to avoid including these libraries when making the build.

Are they excluded somewhere else? Eg when making the build does the Java build engine exclude unused files?

Reply:
analyticsImplementation means gradle assembleAnalytics will include Google libraries, gradle assembleNoAnalytics won't include.
For Chinese app market, we can use noAnalytics build, am I right?
[8:34 AM]
https://riptutorial.com/android/example/402/adding-product-flavor-specific-dependencies
Android Tutorial => Adding product flavor-specific dependencies
Learn Android - Adding product flavor-specific dependencies
[8:34 AM]
This is an example.